app.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+"""Task API Application
+
+This script sets up and runs a Flask-based API for managing tasks.
+"""
+from os import environ
+
+import connexion
+from dotenv import load_dotenv
+
+from login import login_blueprint
+from task import task_blueprint, db
+
+
+def create_app():
+    app = connexion.FlaskApp(__name__)
+    app.add_api('swagger.yaml')
+    application = app.app
+    load_dotenv()
+    application.config['SQLALCHEMY_DATABASE_URI'] = environ.get('DB_URL')
+    application.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = True
+    db.init_app(application)
+
+    with application.app_context():
+        db.create_all()
```

Comment:
@YanaRudyi It is more common to use migrations separately from the application run, you could remove it from here and write instruction in README DSM procedure how to run migrations both for newly provisioned database and upgrading revisions of existing DDL structures